readme.md

```diff
@@ -110,6 +110,7 @@ Colors can be changed by using [`zstyle`](http://zsh.sourceforge.net/Doc/Release
 - `path` (blue) - The current path, for example, `PWD`.
 - `prompt:error` (red) - The `PURE_PROMPT_SYMBOL` when the previous command has *failed*.
 - `prompt:success` (magenta) - The `PURE_PROMPT_SYMBOL` when the previous command has *succeded*.
+- `prompt2:faded` (242) - The color for showing the state of the parser in the continuation prompt (PS2).
```

Comment:
How about `prompt:continuation`?

Reply:
The idea here was that we're already signaling it's the continuation prompt by putting it under `prompt2:` (PS2) instead of `prompt:`. The `faded` means it's the faded part _in front_ of the actual prompt indicator (`❯`), we don't want to indicate we're changing the prompt indicator color in any way. It's mostly similar to `virtualenv` in terms of positioning / look, but we can't really piggy-back on that color, unfortunately.

I'm open to alternative naming suggestions, but I'd like to keep the `prompt2:` separation. It's just that in that scenario `continuation` doesn't really make sense. In the future we might also make `prompt4` (debug) colors tweakable and having everything under one `prompt:` could get messy.